DeeployTest/testUtils/core/execution.py

```
@@ -0,0 +1,220 @@
+
+    if config.simulator == 'banshee':
+        if config.verbose == 1:
+            env["BANSHEE_LOG"] = "warn"
+        elif config.verbose == 2:
+            env["BANSHEE_LOG"] = "info"
+        elif config.verbose >= 3:
+            env["BANSHEE_LOG"] = "debug"
+
+    log.debug(f"[Execution] Simulation command: {' '.join(cmd)}")
+
+    result = subprocess.run(cmd, capture_output=True, text=True, env=env)
```

Ruff (0.14.11)

`subprocess` call: check for execution of untrusted input

(S603)